tox.ini

```
@@ -132,7 +132,7 @@ commands =
 [testenv:check_isort]
 skip_install = True
 deps = isort
```

```suggestion
deps = isort==5.0.3
```

I'm glad we're not pinning the version to v4. However, I would advise pinning the version here now so that we don't hit any more surprises like this again in the future :)

---

Will do, but I do wonder why we never did this.